### ISSUE TITLE
consecutive `mvn install` failes due to some shading behavior

### ISSUE THREAD

**Issue by nlu90** [apache/pulsar#11083]
**Describe the bug**
The compilation fails due to some previous compilation output. Here is the error message:
```
[ERROR] Failed to execute goal org.apache.maven.plugins:maven-shade-plugin:3.2.4:shade (default) on project pulsar-presto-connector-original: Error creating shaded jar: duplicate entry: META-INF/services/org.apache.pulsar.shade.org.glassfish.jersey.internal.inject.InjectionManagerFactory -> [Help 1]
[ERROR]
[ERROR] To see the full stack trace of the errors, re-run Maven with the -e switch.
[ERROR] Re-run Maven using the -X switch to enable full debug logging.
[ERROR]
[ERROR] For more information about the errors and possible solutions, please read the following articles:
[ERROR] [Help 1] http://cwiki.apache.org/confluence/display/MAVEN/MojoExecutionException
[ERROR]
[ERROR] After correcting the problems, you can resume the build with the command
[ERROR]   mvn <args> -rf :pulsar-presto-connector-original
```

**To Reproduce**
run `mvn install -DskipTests` without `clean`

**Expected behavior**
`mvn install` should be successful.

**Screenshots**

**Desktop (please complete the following information):**

**Additional context**

**Comment by codelipenghui** [comment]
The issue had no activity for 30 days, mark with Stale label.

**Comment by tisonkun** [comment]
Duplicate of #17047

Fixed by #17659
